Source/Core/VideoBackends/D3D/D3DShader.cpp

```diff
@@ -177,51 +174,43 @@ bool CompilePixelShader(const std::string& code, D3DBlob** blob, const D3D_SHADE
                D3D::PixelShaderVersionString(), (const char*)errorBuffer->GetBufferPointer());
 
     *blob = nullptr;
-    errorBuffer->Release();
   }
   else
   {
-    *blob = new D3DBlob(shaderBuffer);
-    shaderBuffer->Release();
+    *blob = new D3DBlob(shaderBuffer.Get());
```

Comment:
Sorry for being repetitive but not sure why we are bothering to pass in a raw pointer, we're upping the ref count on the new smart ptr and deallocating the smart ptr in this scope...